tasks/go.py

```diff
@@ -223,8 +223,10 @@ def staticcheck(ctx, targets, build_tags=None, arch="x64"):
 
     tags = build_tags or get_default_build_tags(build="test", arch=arch)
     # these two don't play well with static checking
-    tags.remove("python")
-    tags.remove("jmx")
+    if "python" in tags:
+        tags.remove("python")
+    if "jmx" in tags:
+        tags.remove("jmx")
```

Comment:
I'm a bit curious about this change. Can you elaborate the need?

Reply:
Thanks for asking, I thought it was because `pkg/util/winutil` and `pkg/util/log` did not have them in their build tags but it turns out it was something uglier: the list was passed by reference to the `staticcheck` function and it was modified, so the second time `staticcheck` was ran the tags were not there :)

I have updated it to make an explicit copy.

Reply:
Opened #7761 to fix this properly, can you have a look and fix the failing tests?